scripts/_aws.sh

```diff
@@ -36,6 +36,9 @@ function _aws_login() {
     local profile_name=$1
     if [[ -z ${profile_name} ]]; then
         uhd aws login uhd-dev
+
+        # If using the auth infra you can enable this profile and comment out the uhd-dev command above.
+        # uhd aws login uhd-auth-dev
```

Comment:
Would be good to work this in properly as either part of the command (somehow) or maybe as a new command (e.g. `aws login` and `aws login-auth` perhaps?).